src/hotspot/share/cds/heapShared.cpp

```diff
@@ -831,6 +838,7 @@ void HeapShared::write_subgraph_info_table() {
     _archived_ArchiveHeapTestClass = array;
   }
 #endif
+  print_stats();
```

Comment:
Perhaps make it conditional on `log_is_enabled(Info, cds, heap)`.

Reply:
Fixed as suggested.